sorting/bubbleSort.js

```diff
@@ -0,0 +1,21 @@
+const bubbleSort = (arr) => {
+
+    if(arr !== null) {
```

Comment:
if there is a condition to run a function then try to exit from the function from that condition instead of wrapping the whole function in it.
```javascript
if(arr || !Array.isArray(arr)) 
  return
```